packages/lodestar/src/chain/attestation.ts

```diff
@@ -57,13 +57,13 @@ export class AttestationProcessor implements IAttestationProcessor {
       return;
     }
     const targetRoot = attestation.data.target.root;
-    if (!await this.db.block.has(targetRoot.valueOf() as Uint8Array)) {
+    if (!this.forkChoice.hasBlock(targetRoot.valueOf() as Uint8Array)) {
```

Comment:
how much blocks does forkchoice hold? all non finalized?

Reply:
yeah basically it stores everything in db.block with (n-1) non finalized and 1 finalized node/block